### PR TITLE
documentation of animating

### PR DESCRIPTION
**Pull request by MeikeWeiss**
I changed the documentation a bit and reorganized the methods.

### CODE REVIEW

File: gap/PolygonalComplexes/animating.gd [hunk 1]
```diff
@@ -34,47 +34,62 @@
...
 #! <Enum>
 #!   <Item>Construct the surface, e.g. the octahedron above.</Item>
 #!   <Item>Configure your animation using the methods in the Sections
 #!   <Ref Subsect="Section_LabelVisibility"/> and <Ref Subsect="Section_LabelColouring"/>.</Item>
-#!   <Item>Write your animation to a file calling the main method <K>DrawSurfaceToJavaScript</K> (see Section <Ref Sect="Section_LabelCoordinatesAndCoreFunctionality"/>).</Item>
+#!   <Item>Write your animation to a file calling the main method <K>DrawSurfaceToJavaScript</K>
+#!        (see Section <Ref Sect="Section_LabelCoordinatesAndCoreFunctionality"/>).</Item>
 #! </Enum>
-#! The output of the method <K>DrawSurfaceToJavaScript</K>(<Ref Subsect="DrawSurfaceToJavaScript"/>) is a html file. The file can be opened with any browser and then the animation is shown directly.
+#! The output of the method <K>DrawSurfaceToJavaScript</K>(<Ref Subsect="DrawSurfaceToJavaScript"/>) is a html file.
```
Comment (TomGoertzen) [inline gap/PolygonalComplexes/animating.gd:54]: should be "an html file"

File: gap/PolygonalComplexes/animating.gd [hunk 2]
```diff
@@ -34,47 +34,62 @@
...
+#! of the surface which should be animated (see <K>SetVertexCoordinates3D</K> <Ref Subsect="SetVertexCoordinates3D"/>).
 #!
 #! After setting the vertex coordinates, the location of the edges in <M>\mathbb{R}^3</M> has to be derived.
-#! If this is not done, the animation becomes inconsistent, as the following example shows.
-#! We animate a tetrahedron, but then we replace all vertex coordinates with their negatives (without recomputing the edge locations).
+#! By default, the method <K>DrawSurfaceToJavaScript</K>(<Ref Subsect="DrawSurfaceToJavaScript"/>)
+#! always calculates the edge locations depending on the current vertices positions.
+#! If you want to avoid that, you can use the method <K>DrawSurfaceToJavaScriptCalculate</K>(<Ref Subsect="DrawSurfaceToJavaScript"/>)
+#! and set the last parameter to false. In this case, only the first call of the method computes the edge locations.
+#! Using this method can makes the animation inconsistent if the coordinates of the vertices have changed but the edge locations have stayed the same.
```
Comment (TomGoertzen) [inline gap/PolygonalComplexes/animating.gd:70]: should be "can make"

File: gap/PolygonalComplexes/animating.gd [hunk 3]
```diff
@@ -239,98 +259,121 @@ DeclareOperation( "CalculateParametersOfEdges", [IsSimplicialSurface, IsRecord]
...
 
-#! @BeginGroup ActivateEdge
+#! @BeginGroup DeactivateEdges
 #! @Description
-#! Activate the edge <A>edge</A>. If an edge is active, then he is shown as a line in the animation. (For more information look at the start of the section <Ref Subsect="Section_LabelVisibility"/>)
-#! @Returns the updated print record.
-#! @Arguments surface, edge, printRecord
-DeclareOperation( "ActivateEdge", [IsSimplicialSurface, IsPosInt, IsRecord] );
-#! @Arguments surface, edge, printRecord
+#! The method <K>DeactivateEdge</K>(<A>surface</A>, <A>i</A>, <A>printRecord</A>) deactivate the edge <A>i</A>.
```
Comment (TomGoertzen) [inline gap/PolygonalComplexes/animating.gd:305]: should be "deactivates"

File: gap/PolygonalComplexes/animating.gd [hunk 4]
```diff
@@ -239,98 +259,121 @@ DeclareOperation( "CalculateParametersOfEdges", [IsSimplicialSurface, IsRecord]
...
 
-#! @BeginGroup ActivateFace
+#! @BeginGroup DeactivateFaces
 #! @Description
-#! Activate the face <A>face</A>. If a face is active, then the face is shown in the animation as an area. (For more information look at the start of the section <Ref Subsect="Section_LabelVisibility"/>)
-#! @Returns the updated print record.
-#! @Arguments surface, face, printRecord
-DeclareOperation( "ActivateFace", [IsSimplicialSurface, IsPosInt, IsRecord] );
-#! @Arguments surface, face, printRecord
+#! The method <K>DeactivateFace</K>(<A>surface</A>, <A>i</A>, <A>printRecord</A>) deactivate the face <A>i</A>.
```
Comment (TomGoertzen) [inline gap/PolygonalComplexes/animating.gd:333]: see above

**Review by TomGoertzen — COMMENTED** [review]
Very nice!!! I made a few comments you might want to check out :)

**Review by TomGoertzen — APPROVED** [review]

**Review by ReymondAkpanya — APPROVED** [review]

**Pull request merged**